db/sovereign-db/src/ledger_db/rpc.rs

```diff
@@ -124,8 +124,8 @@ impl LedgerRpcProvider for LedgerDB {
         tx_ids: &[sovereign_sdk::rpc::TxIdentifier],
         _query_mode: QueryMode,
     ) -> Result<Vec<Option<Self::TxResponse>>, anyhow::Error> {
-        // TODO: Sort the input and use an iterator instead of querying for each slot individually
-        // https://github.com/Sovereign-Labs/sovereign/issues/191
+        // TODO: https://github.com/Sovereign-Labs/sovereign/issues/191 Sort the input
+        //      and use an iterator instead of querying for each slot individually
```

Comment:
Space is added so IDE shows second line when you search for TODOs. I hope it is not annoying